openedx/features/edly/api/tests/test_serializers.py

```diff
@@ -40,27 +45,57 @@ def setUp(self):
             'SITE_NAME': self.edly_sub_org_of_user.lms_site.domain,
             'course_org_filter': self.edly_sub_org_of_user.edx_organization.short_name,
         }
-        SiteConfigurationFactory(
+        self.site_configuration = SiteConfigurationFactory(
             site=self.edly_sub_org_of_user.lms_site,
             enabled=True,
             values=self.test_site_configuration,
         )
 
+    def validate_url(self, url):
+        """
+        Validates a given string as url
+        """
+        validate = URLValidator()
+        try:
+            validate(url)
+        except ValidationError:
+            return False
+        return True
+
+    def get_expected_url(self):
```

Comment:
`get_expected_host_url`

Reply:
@zubair-arbi This is generic function used for `site_url` and `host_url` both.